gammapy/spectrum/spectrum_analysis.py

```diff
@@ -480,11 +668,38 @@ def model(self, model, name=None):
                 raise ValueError("Undefined model string: {}".format(model))
 
         if not isinstance(model, sherpa.models.ArithmeticModel):
-            raise ValueError("Only sherpa models are supported at the moment")
+            raise ValueError("Only sherpa models are supported")
 
         self._model = model
 
     @property
+    def statistic(self):
```

Comment:
In `__init__` you called the data member `statistic`.
Now you're creating a property `statistic` that accesses a data member `_statistic`.
Is this a bug?